lib/rules/no-underscore-dangle.js

```diff
@@ -125,6 +130,30 @@ module.exports = {
                 node.object.object.type === "ThisExpression";
         }
 
+        /**
+         * Check if function parameter has a underscore at the beginning.
+         * @param {ASTNode} node node to evaluate
+         * @returns {boolean} true if function parameter name has a underscore at the beginning.
+         * @private
+         */
+        function checkForFollowingUnderscoreInFunctionParameter(node) {
```

Comment:
Can we change this to `checkForTrailingUnderscoreInFunctionParameters` (or `checkForDanglingUnderscoreInFunctionParameters` if you're will to change all the other function names as well) to match the other method names?